config/initializers/darksky.rb

```diff
@@ -0,0 +1,3 @@
+require 'forecast_io'
+# could also use ENV['forecast_io_key']
+ForecastIO.api_key = Figaro.env.forecast_io_key
```

Comment:
@derekriemer now that this code is in an initializer we won't have to call or require it anywhere.  It gets setup before the classes in our application are loaded.